graphql_compiler/compiler/expressions.py

```diff
@@ -392,6 +411,21 @@ def to_gremlin(self):
 
         return template.format(mark_name=mark_name, field_name=field_name)
 
+    def to_cypher(self):
+        """Return a unicode object with the Cypher representation of this expression."""
+        self.validate()
+
+        mark_name, field_name = self.location.get_location_name()
+        if field_name is not None:
```

Comment:
Nit: Add a newline between 418 and 419 for consistency between the to_match and to_gremlin methods.